src/Aspire.Dashboard/Model/ResourceViewModel.cs

```diff
@@ -73,6 +73,29 @@ public enum ReadinessState
     Ready
 }
 
+public sealed class ResourceViewModelNameComparer : IComparer<ResourceViewModel>
```

Comment:
internal?

Reply:
Doesn't matter. No one should reference the dashboard